components/date-time/time.js

```diff
@@ -122,43 +188,102 @@ class TimePicker extends Component {
 
 	render() {
 		const { is12Hour } = this.props;
-		const { minutes, hours, am } = this.state;
+		const { day, month, year, minutes, hours, am } = this.state;
 
 		return (
 			<div className="components-time-picker">
-				<input
-					className="components-time-picker__input"
-					type="text"
-					value={ hours }
-					onChange={ this.onChangeHours }
-					onBlur={ this.updateHours }
-				/>
-				<span className="components-time-picker__separator">:</span>
-				<input
-					className="components-time-picker__input"
-					type="text"
-					value={ minutes }
-					onChange={ this.onChangeMinutes }
-					onBlur={ this.updateMinutes }
-				/>
-				{ is12Hour && <div>
-					<Button
-						isDefault
-						className="components-time-picker__am-button"
-						isToggled={ am === 'AM' }
-						onClick={ this.updateAmPm( 'AM' ) }
-					>
-						{ __( 'AM' ) }
-					</Button>
-					<Button
-						isDefault
-						className="components-time-picker__pm-button"
-						isToggled={ am === 'PM' }
-						onClick={ this.updateAmPm( 'PM' ) }
-					>
-						{ __( 'PM' ) }
-					</Button>
-				</div> }
+				<fieldset key="date-time-form" className="date-time-form">
+					<div key="month-field" className="input-container month">
+						<select
+							id="date-time-form--month-field"
+							aria-labelledby="Month"
+							name="month"
+							value={ month }
+							onChange={ this.onChangeMonth }
+							onBlur={ this.updateMonth }>
```

Comment:
Just a nitpick but should the `>` go on a newline? It might improve readability; if that violates our `eslint` my bad. I'm still getting used to Gutenberg's style 😄

Reply:
Yeah, same here. Better nitpick sooner than later!

Reply:
Fix'd!